Tests/Expected/Colors/default-context-defaults.swift

```diff
@@ -56,12 +57,9 @@ enum ColorName {
     return Color(named: self)
   }
 }
-// swiftlint:enable type_body_length
```

Comment:
I'm not sure we shouln't re-enable this rule.

You don't know how a template will change over time, and we disable rules only when necessary. In this case, we only disable it for the enum because it can have an arbitrary length.